src/state.py

```diff
@@ -616,6 +627,16 @@ def populate_players_stats(self):
             for c in self.passes[p]:
                 self.players[p].passes.update({c: self.passes[p][c]})
 
+    def recompute_frame_count(self):
+        "recompute frame count of all players in frames"
```

Comment:
Adjust documentation to reflect that this function computes the total number of frames each player_id appears in the video ==> used potentially to adjust for noise in reassigned ids